content/docs/tasks/security/authn-policy/index.md

```diff
@@ -570,6 +570,9 @@ $ for i in `seq 1 10`; do curl --header "Authorization: Bearer $TOKEN" $INGRESS_
 401
 {{< /text >}}
 
+JWT policy could also be setup on `istio-ingressgateway`, see the
```

Comment:
You can add a JWT policy on `istio-ingressgateway`. This is often used in the scenario that you want to define JWT policies that apply at the Gateway level, instead of at individual services. See the ... for more details.